auth/resources/keystore.properties.sample

```diff
@@ -24,14 +24,3 @@ s3KeyStorePassword=seagate
 s3KeyPassword=seagate
 s3AuthCertAlias=s3auth_pass
 s3CipherUtil=s3cipher generate_key --const_key cortx
-
-# Uncomment this to override logging config file.
-#
-# logConfigFile=<PATH TO CONFIG FILE>
-
-# Uncomment this to override log level.
-# Note - setting this will override the logging level of root logger.
-# If this option is set along with logConfigFile, the log level set in the
-# config file will be overridden.
-#
-logLevel=INFO
```

Comment:
What will be default loglevel?if we don;t have this entry itself?and we don;t need to manipulate loglevel?

Reply:
the default log level will be "INFO" only and it can be changed from authserver.properties and there is no change in this .
We dont need entry in keystore.properties ..hence i have removed this form keystore.properties